src/Bicep.LangServer/BicepCompilationManager.cs

```diff
@@ -235,14 +234,14 @@ private ImmutableArray<ISourceFile> CloseCompilationInternal(DocumentUri documen
                             }
                         }
 
-                        if (reloadBicepConfig)
+                        var configuration = reloadBicepConfig
```

Comment:
Reverting a change that was made in the commit mentioned in this PR description(https://github.com/Azure/bicep/commit/873075cac432d05a37be7cd00c42e6f1f1589688).